core/.js/src/main/scala/io/udash/bindings/inputs/Input.scala

```diff
@@ -7,26 +7,27 @@ import scalatags.JsDom.all._
 
 import scala.concurrent.duration.{Duration, DurationInt}
 
-/** Abstraction for HTML input tags.*/
+/** Abstraction for HTML input tags. */
 private[bindings] abstract class Input(inputType: String) {
   /**
-    * @param value Property to bind.
-    * @param debounce Property update timeout after input changes.
-    * @param inputModifiers Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
-    * @return HTML input with bound Property, applied modifiers and nested options.
-    */
-  def apply(value: Property[String], debounce: Duration = 20 millis)(inputModifiers: Modifier*): InputBinding[JSInput] =
+   * @param value               Property to bind.
+   * @param debounce            Property update timeout after input changes.
+   * @param onPropertyUpdated   Callback that's being called when property is updated
```

Comment:
These docs are not conveying the difference between property update and input value update. I'm actually considering if `onPropertyUpdated` is what you want to achieve here long term, or we should use another name. Note that this is an externally passed property, you're not executing the callback when someone changes the value outside of the input.

nit: why is it `being called` and not just `called`?
nit: add missing full stop
nit: you can avoid repetition by changing `called` to `executed`